triggers/tests/src/test/scala/com/digitalasset/daml/lf/engine/trigger/test/ConfigSpec.scala

```diff
@@ -102,7 +98,7 @@ class ConfigSpec
     "succeed for user with primary party & actAs and readAs claims" in {
       for {
         client <- LedgerClient(channel, clientConfig)
-        userId = randomUserId()
+        userId = randomUserId
```

Comment:
Parentheses were indicating that the function has side effects. It is still the case, so I would add them back, alongside with modifying `SandboxRequiringAuthorizationFuns`